screen2.0/src/app/applets/multi-region-search/page.tsx

```diff
@@ -1,12 +1,10 @@
 "use client"
 import { Typography } from "@mui/material"
-import BedUpload from "../../../common/components/BedUpload"
 
 export default function MultiRegionSearch() {
   return (
     <main>
-      <Typography variant="h6" mt={2}>Find Intersecting cCREs from BED file</Typography>
-      <BedUpload />
+      <Typography variant="h6" mt={2}>This is the Multi Region Search Page</Typography>
```

Comment:
Shall we remove multi region search from applets since we have added it search ?

Reply:
I think I remember Jill saying that in the future we could have something similar to bed intersect but with more features. I don't remember exactly, I'll ask

Reply:
sounds good